src/api/SequenceNumber/useSequenceNumber.ts

```diff
@@ -0,0 +1,43 @@
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { subWebContext } from "src/api/SPWebContext";
+import { z } from "zod";
+import "@pnp/sp/items";
+
+const SequenceOffset = z.array(
+  z.object({
+    Id: z.number().int().positive(),
+    Title: z.number().int().positive(),
+  })
+);
+
+type SequenceOffset = z.infer<typeof SequenceOffset>;
+
+const useSequenceOffset = (program: string) => {
+  return useQuery({
+    queryKey: ["SequenceOffset", program],
+    queryFn: () =>
+      subWebContext(program)
+        .web.lists.getByTitle("SequenceOffset")
+        .items.top(1)
+        .orderBy("Id", false)(),
+    select: (data: SequenceOffset) => SequenceOffset.parse(data),
```

Comment:
Not sure it really matters much. But the type of incoming data is slightly different than the type specified here, as the Title is a string not a number.